tests/test_service.py

```diff
@@ -227,9 +227,6 @@ def test_get_wishlist_not_found(self):
         resp = self.app.get("/wishlists/0")
         self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
         data = resp.get_json()
-        self.assertEqual(data['error'], "Not Found")
-        self.assertEqual(data['message'], ("404 Not Found:"
```

Comment:
We can edit the message to be checked rather than removing it.

Reply:
Done.